Kernel/kprintf.cpp

```diff
@@ -7,7 +7,10 @@
 #include <AK/PrintfImplementation.h>
```

Comment:
Maybe it is then possible to merge `Kernel/kprintf.cpp` and `Kernel/Arch/aarch64/kprintf.cpp`?

Reply:
Not yet, but it will be possible soon.